README_PYPI.rst

```diff
@@ -1,25 +1,48 @@
 .. # README file for Pypi
 
-Pywbemtools is a collection of command line tools  is a WBEM client,
-written in pure Python. It supports Python 2 and Python 3.
+Pywbemtools is a collection of command line tools that communicate with WBEM
+Servers. The tools are written in pure Python and support Python 2 and Python
+3.
 
-A WBEM client allows issuing WBEM operations to a WBEM server, using the
-`CIM/WBEM standards`_ defined by the DMTF, for the purpose of performing
-systems management tasks. A WBEM indication listener is used to wait for
-and process notifications emitted by a WBEM server, also for the purpose
-of systems management.
+Pywbemtools provides a command line tool (pywbemcli) that uses the `pywbem`_
+Python WBEM client infrastructure to issue operations to a WBEM server using
+the `CIM/WBEM standards`_ defined by the `DMTF`_ to perform system management
+tasks.
 
-In addition to providing a command line interface for the client side DMTF defined
-WBEM Operations, pywbem cli provides commands for a number of high level
-operations that are useful to developers and users of WBEM Servers such
-as the implementation of the SNIA SMI specification.
+CIM/WBEM infrastructure is used for a wide variety of systems management tasks
+in the industry including DMTF management standards and  the `SNIA`_
+Storage Management Initiative Specification `SMI-S`_.
 
-CIM/WBEM infrastructure is used for a wide variety of systems management
-tasks in the industry.
+Pywbemcli provides the functionality to :
+
+* Explore the CIM data of WBEM Servers.  It can manage/inspect the CIM
+  model components including CIM classes, CIM instances, and CIM
+  qualifiers and execute CIM methods and queries on the WBEM server. It
+  implements subcommands to execute the CIM-XML operations defined in the
+  DMTF specification
+  [CIM Operations Over HTTP'(DSP0200)](https://www.dmtf.org/sites/default/files/standards/documents/DSP0200_1.4.0.pdf)).
+
+* Inspect/manage WBEM server functionality such as CIM namespaces, WBEM
+  registered profiles, and other server information.
+
+* Capture detailed information on interactions with the WBEM server
+  including time statistics.
+
+For more information on pywbemtools see the `pywbemtools readme`_ and the
+`pywbemtools documentation`_
+
+.. _pywbemtools readme: https://github.com/pywbem/pywbemtools/blob/master/README.rst
+.. _pywbemtools documentation: https://pywbemtools.readthedocs.io/en/stable/
 
 For more information on pywbem, see the `pywbem readme`_, and the
 `pywbem documentation`_.
 
-.. _pywbem readme: https://github.com/pywbem/pywbemtools/blob/stable_0.14/README.rst
-.. _pywbem documentation: https://pywbemtools.readthedocs.io/en/stable_0.6/
+.. _pywbem: https://github.com/pywbem/pywbem/blob/stable_0.14/README.rst
```

Comment:
This URL still contains a pywbem version. Since this is the general link to pywbem, I suggest using the github repo link: https://github.com/pywbem/pywbem.

Reply:
Actually there were two issues. The one above and the one with the ref to HTTP Operations over HTTP. **DONE. Both Fixed**